src/main/java/reposense/RepoSense.java

```diff
@@ -45,4 +50,47 @@ public static void main(String[] args) {
             logger.log(Level.WARNING, pe.getMessage(), pe);
         }
     }
+
+    /**
+     * Gets a list of {@code RepoConfiguration} based on {@code cliArguments}'s subclass type.
+     *
+     * @throws IOException if user-supplied csv file does not exists or is not readable.
+     */
+    public static List<RepoConfiguration> getRepoConfigurations(CliArguments cliArguments) throws IOException {
+        if (cliArguments instanceof ConfigCliArguments) {
```

Comment:
seems like my previous comment did not go through for this one

Reply:
I have moved it to `RepoSense#main` but refactored to `RepoSense#getRepoConfigurations` to do the `instanceof` checks.

May I know what is missing?